stdlib/transactions/src/main/ballerina/transactions/commons.bal

```diff
@@ -289,11 +299,13 @@ function getParticipant2pcClient(string participantURL) returns Participant2pcCl
     }
 }
 
-# Registers a participant with the initiator's coordinator. This function will be called by the participant
+# Registers a participant with the initiator's coordinator. This function will be called by the participant.
 #
-# + transactionId - - ID of the transaction to which this participant is registering with
-# + transactionBlockId - - The local ID of the transaction block on the participant
-# + registerAtURL - - The URL of the initiator to which this participant will register with
+# + transactionId - ID of the transaction to which this participant is registering with.
```

Comment:
Shall we change this as " Global transaction ID to which this participant is registering with"